core/domain/exp_domain.py

```diff
@@ -1302,6 +1306,45 @@ def is_demo(self):
         """
         return self.is_demo_exploration_id(self.id)
 
+    # This property is only meant to be used for oppia-android.
+    # This property gives the information about the proto size in bytes of the
+    # exploration, which is used in the oppia-android application.
+    @property
+    def android_proto_size_in_bytes(self):
+        """Returns the most up-to-date size of the exploration proto,
+         recomputing from scratch if necessary.
+
+        Returns:
+            int. Updated exploration's proto size, in bytes.
```

Comment:
int. The size of the exploration's Android proto representation, in bytes.

Reply:
Updated